cirq/circuits/circuit.py

```diff
@@ -1503,8 +1503,21 @@ def with_device(
             device=new_device,
         )
 
+    @deprecated_parameter(
+        deadline='v0.10.0',
```

Comment:
Should be 
```suggestion
        deadline='v0.11.0',
```
We'll be releasing v0.10 soon - that's the current dev version

Reply:
done